funcx_endpoint/funcx_endpoint/tests/strategies/test_kube_simple.py

```diff
@@ -0,0 +1,160 @@
+import threading
+import time
+from pytest import fixture
+
+from funcx_endpoint.executors.high_throughput.interchange import Interchange
+from funcx_endpoint.strategies.kube_simple import KubeSimpleStrategy
+
+
+def no_op_worker():
+    """
+    This is used to trick the base strategy into not doing scheduled checks.
+    The strategize calls are only triggered by explicit calls to make_callback in the
+    tests.
+    :return:
+    """
+    pass
+
+
+class TestKubeSimple:
+
+    @fixture
+    def mock_interchange(self, mocker):
+        mock_interchange = mocker.MagicMock(Interchange)
```

Comment:
Not quite familiar with mocker. But does mocker need some imports here?

Reply:
`mocker` is passed in as a [pytest fixtur](https://docs.pytest.org/en/stable/fixture.html)